### PR TITLE
Allow installing Lönn instead of Ahorn in Olympus

### PR DESCRIPTION
**Pull request by max4805**
Still heavily WIP, since I did not test it on Linux and need to mess around to make it work on macOS.

### CODE REVIEW

**Comment by max4805** [comment]
This should be ready! Here is what it looks like:

[Olympus 2022-12-11 17-27-18.mp4.webm](https://user-images.githubusercontent.com/52103563/206916497-0ed5d3ec-d8be-4a76-b32c-01c9b9298f4b.webm)

**Comment by swoolcock** [comment]
C# looks good.  OlympUI is still a little foreign to me so I can't really comment on that part.

File: src/scenes/mainmenu.lua [hunk 1]
```diff
@@ -287,6 +287,170 @@ Press the manage button below.]])
...
+        local downloadLink = data.Item2
+
+        -- version info
+        local content = {
+            uie.label(
+                (config.loennInstalledVersion == ""
+                    and "Lönn is currently not installed."
+                    or "Currently installed version: " .. config.loennInstalledVersion) .. "\n" ..
+                "Latest version: " .. latestVersion .. "\n" ..
+                "Install folder: " .. config.loennRootPath
```
Comment (Cruor) [inline src/scenes/mainmenu.lua:338]: Could be made a bit more readable with a `string.format` or `table.concat` on lines? Works just fine as it is though.

**Review by Cruor — APPROVED** [review]
Only looking at the Lua changes, looks good to me, just a small nitpick.

**Pull request merged**